exporter/exporter.go

```diff
@@ -392,11 +392,16 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 		"stream_group_consumer_idle_seconds":           {txt: `Consumer idle time in seconds`, lbls: []string{"db", "stream", "group", "consumer"}},
 		"stream_group_consumer_messages_pending":       {txt: `Pending number of messages for this specific consumer`, lbls: []string{"db", "stream", "group", "consumer"}},
 		"stream_group_consumers":                       {txt: `Consumers count of stream group`, lbls: []string{"db", "stream", "group"}},
+		"stream_group_entries_read":                    {txt: `Total number of entries read from the stream group`, lbls: []string{"db", "stream", "group"}},
+		"stream_group_lag":                             {txt: `The number of messages waiting to be delivered to the stream group's consumers`, lbls: []string{"db", "stream", "group"}},
 		"stream_group_last_delivered_id":               {txt: `The epoch timestamp (ms) of the last delivered message`, lbls: []string{"db", "stream", "group"}},
 		"stream_group_messages_pending":                {txt: `Pending number of messages in that stream group`, lbls: []string{"db", "stream", "group"}},
 		"stream_groups":                                {txt: `Groups count of stream`, lbls: []string{"db", "stream"}},
 		"stream_last_generated_id":                     {txt: `The epoch timestamp (ms) of the latest message on the stream`, lbls: []string{"db", "stream"}},
 		"stream_length":                                {txt: `The number of elements of the stream`, lbls: []string{"db", "stream"}},
+		"stream_max_deleted_entry_id":                  {txt: `The epoch timestamp (ms) of last message was deleted from the stream`, lbls: []string{"db", "stream"}},
```

Comment:
This and the next two fields all have "entry_id" in the name yet the description talks about "epoch timestamp (ms)" - which one is it? A timestamp or an ID? Let's get the naming right and put the correct base unit in the name of the metric ([see prom docs here](https://prometheus.io/docs/practices/naming/))

Reply:
Great question, I was using what was defined by `stream_last_generated_id` as the baseline for these defintions. Since the ID is an epoch time, so using first and last you can understand how long the stream data exists for and the `stream_max_deleted_entry_id` is for understanding which specific entry was removed from the stream when `XDEL` is used.

If the description should be changed, then we should probably update the description for `stream_last_generated_id` also.

Reply:
Ok, then lets leave it as is, thanks for explaining!